net-dialup/linux-atm/files/linux-atm-2.5.2-c99-musl.patch

```diff
@@ -0,0 +1,120 @@
+Author: NHOrus <jy6x2b32pie9@yahoo.com>
+Bug: https://bugs.gentoo.org/897842
+Enabling the system extension that gate some POSIX features
+and fixing missing includes, 32/64 bit confusion, standard
+atexit function instead of non-standard, and function signature
+diff -pur a/configure.in b/configure.in
+--- a/configure.in
++++ b/configure.in
+@@ -34,6 +34,7 @@ dnl We have some special PERL scripts wh
+ AC_PATH_PROG(PERL, perl)
+ AC_SUBST(PERL)
+ 
++AC_USE_SYSTEM_EXTENSIONS
+ 
+ dnl Check for needed header files
+ AC_CHECK_HEADER(asm/errno.h, ,
+diff -pur a/src/lib/unix.c b/src/lib/unix.c
+--- a/src/lib/unix.c
++++ b/src/lib/unix.c
+@@ -10,6 +10,7 @@
+ #include <stdio.h>
+ #include <unistd.h>
+ #include <errno.h>
++#include <string.h>
+ #include <sys/types.h>
+ #include <sys/stat.h>
+ #include <sys/socket.h>
+diff -pur a/src/sigd/policy.c b/src/sigd/policy.c
+--- a/src/sigd/policy.c
++++ b/src/sigd/policy.c
+@@ -8,6 +8,7 @@
+ 
+ #include <atm.h>
+ #include <atmd.h>
++#include <string.h>
+ 
+ #include "proto.h" /* for "pretty" */
+ #include "policy.h"
+--- a/src/sigd/kernel.c	2024-04-27 11:36:41.203470908 +0000
++++ b/src/sigd/kernel.c	2024-04-27 11:37:36.533130285 +0000
```

Comment:
You can use `scrub-patch` from `app-portage/iwdevtools` to get rid of the timestamps and some extra stuff.  There are a few pointers in the [devmanual](https://devmanual.gentoo.org/ebuild-writing/misc-files/patches/#clean-patch-howto)